README.md

```diff
@@ -44,4 +44,39 @@ pip install zep-python
 
 ```bash
 poetry add zep-python
-```
\ No newline at end of file
+```
+
+## Zep cloud Installation
+In order to install zep python sdk with zep-cloud support, you will need to install
+a release candidate version.
+
+```bash
+pip install --pre zep-python
+```
+
+-- OR --
+
+```bash
+poetry add zep-python@^2.0.0-rc
+```
+
+You will also need to provide a Zep Project API key to your zep client for cloud support.
+You can find out about zep projects in our [cloud docs](https://help.getzep.com/projects.html)
```

Comment:
capitalization: `Zep`. and when referring to `Projects` in Zep, they're a proper noun and capitalized.  Please review use elsewhere for consistency.